yamscripts.yml

```diff
@@ -10,9 +10,10 @@ scripts:
 
 # bundle: Bundle assets for production
   bundle:
-    static: cpx "content-src/static/**/*" data/content -C
+    static: cpx "content-src/static/**/!(activity-streams.html)" data/content -C
```

Comment:
Ah! got it. I would remove the `-C` and add a `clean` task instead that runs synchronously